### PR TITLE
Mccalluc/tsv lineup

### LINKED ISSUE #2040: Ordino/lineup for metadata
Building on #2039 (lots of questions there about scope, column naming, etc.), we would like to make it easy for users to understand the data.

@ngehlenborg : The is [no documentation](https://github.com/Caleydo/ordino) on how Ordino would be used as a component. Would it be ok to try the [React wrapper for Lineup](https://github.com/lineupjs/lineupjsx) for now?

### PR DESCRIPTION
**Pull request by mccalluc**
Draft: Would fix #2040 

@john-conroy : You've had concerns in the past about bundle size / dependencies. With the JS lazy loading, it's really only the docker image size that's a concern, right, or am I missing something? If it is just the docker image, can you clarify what bottlenecks you see us hitting, and depending on that, whether you'd have reservations about merging this?

@ngehlenborg : I can't say that it really works out of the box. 
- No rows are displayed.
- Many more columns than space allows: Should we manually specify a subset? See if the div can scroll horizontally?
- Controls in the lower right don't seem to work.
- What's the best way to move this forward: Are folks in Austria interested in getting this working for us? Or is it on our shoulders? 
- Is there a non-CDN way of pulling the necessary CSS?
Screen Shot 2021-08-31 at 1 12 51 PM

### CODE REVIEW

**Comment by mccalluc** [comment]
Rendering fixed. Donors and Samples works... Datasets may have a race condition? Filed https://github.com/lineupjs/lineupjsx/issues/27. I don't think that should block this, particularly if this is not intended for public use.

**Review by john-conroy — COMMENTED** [review]
Just looking at the diffs, let me know if you want me to take a look at it locally. Maybe before merging inspect the bundle to make sure everything is being split correctly?

File: context/app/routes_api.py [hunk 1]
```diff
@@ -1,26 +1,47 @@
...
 
 @blueprint.route('/api/v0/<entity_type>.tsv')
 def entities_tsv(entity_type):
+    entities = _get_entities(entity_type)
+    return _make_tsv_response(_dicts_to_tsv(entities, _first_fields), f'{entity_type}.tsv')
+
+
+@blueprint.route('/lineup/<entity_type>')
+def lineup(entity_type):
+    entities = _get_entities(entity_type)
```
[the remark this answers is not in the document]
Comment (mccalluc) [reply context/app/routes_api.py:20]: good point... for now, my sense is that this is internal: If it gets wider usage, then we should set up a JSON endpoint to complement the TSV, and hit that from the client side... but for now I don't want to add that complexity.

**Comment by mccalluc** [comment]
Feedback on CSS and section container addressed. Good enough to merge.

**Pull request merged**